### DISCUSSION TITLE
Bosh for SAP Appliance

### DISCUSSION THREAD

**Discussion by AmauryVanEspen** [cloudfoundry/bosh#2416] [Q&A]
Hello,
I would like to know if BOSH is relevant to deploy SAP Appliance on top of OpenStack
for example, i've found an archive of SAP for kubernetes here :  SAP/bosh-kubernetes-cpi-release
Any resource would be useful to better understand the way to deploy appliance on private cloud.
Thank you
Amaury

**Comment by beyhan** [comment]
Hi @AmauryVanEspen,
In general BOSH is a generic tool for release engineering, deployment, and lifecycle management which could be used to deploy software to supported IaaS providers. OpenStack is one of them. There is nothing SAP specific in BOSH.  SAP/bosh-kubernetes-cpi-release was an attempt to implement BOSH CPI for K8s but it didn't get any adoption and the project is deprecated. What do you mean with SAP Appliance? Does this answer your question?

**Comment by tanerjn** [comment]
Hello @AmauryVanEspen. Check the template I am sharing for OpenStack. You of course have to use your SAP stack and tweak few things for compliance.  
https://github.com/tanerjn/openstack-deployment

**Comment by AmauryVanEspen** [comment]
Dear @tanerjn

Thank you for your fast answer.
There is a related research paper
https://www.researchgate.net/publication/325381449_Cloud_deployment_continuous_integration_of_5G-CogNet_components
nice !

Could we have a look a bit deeper together to the related SAP stack for
tweaking ? Thanks.

Amaury

Le mar. 3 janv. 2023 à 23:36, Taner Metin ***@***.***> a
écrit :

**Reply by tanerjn** [reply]
@AmauryVanEspen Sure. HMU at tanrmetn@gmail.com

I am sure we can come up with something working for you.

**Comment by AmauryVanEspen** [comment]
Dear Began Veli,
SAP provides preconfigured virtual images as a bundle in kind of VM
appliance.
It has to be deployed on cloud.
I would like to deploy it on private cloud hosted on OpenStack. I believed
that BOSH could help in performing this according to some posts I've read.
Thanks
Amaury

Le mar. 3 janv. 2023 à 09:04, Beyhan Veli ***@***.***> a
écrit :
